mypy `error: "CairoContext" has no attribute "set_source_rgba"  [attr-defined]` et al.

I'm using:
```
python 3.12
pygobject 3.54.5 (Gtk4.0)
pygobject-stubs 2.14.0
mypy 1.18.2
pycairo 1.28.0
```

I get errors for every call I make to a [`Gdk.CairoContext`](https://api.pygobject.gnome.org/Gdk-4.0/class-CairoContext.html) method. Specifically, `mypy` reports these errors:

```
error: "CairoContext" has no attribute "fill"  [attr-defined]
error: "CairoContext" has no attribute "line_to"  [attr-defined]
error: "CairoContext" has no attribute "move_to"  [attr-defined]
error: "CairoContext" has no attribute "rectangle"  [attr-defined]
error: "CairoContext" has no attribute "set_source_rgba"  [attr-defined]
```

All of my other `PyGObject` code passes `mypy` checks and my code works. I'm using the approach in the [`Gtk.DrawingArea`](https://api.pygobject.gnome.org/Gtk-4.0/class-DrawingArea.html) example and every other example I've found online. The first tutorial I worked with was at [Custom drawing area using Cairo](https://github.com/Taiko2k/GTK4PythonTutorial?tab=readme-ov-file#custom-drawing-area-using-cairo) from [Taiko's GTK4 Python tutorial](https://github.com/Taiko2k/GTK4PythonTutorial). 

### pygobject.stubs references

There is a [`CairoContext`](https://github.com/pygobject/pygobject-stubs/blob/d36752ab704eedfca6a53f447cdb3e5d7ca5bc77/src/gi-stubs/repository/_Gdk4.pyi#L2611) class in `_Gdk4.pyi`, and it is missing those method calls.

The class `cairo.Context` is a type hint in 9 files.

The c versions of two of the methods show up in ``_Gdk4.pyi``, with `cairo_` prepended and a cairo.Context as the first argument:
[`def cairo_rectangle(cr: cairo.Context[_SomeSurface], rectangle: Rectangle) -> None: ...`](https://github.com/pygobject/pygobject-stubs/blob/d36752ab704eedfca6a53f447cdb3e5d7ca5bc77/src/gi-stubs/repository/_Gdk4.pyi#L2489)
[`def cairo_set_source_rgba(cr: cairo.Context[_SomeSurface], rgba: RGBA) -> None: ...`](https://github.com/pygobject/pygobject-stubs/blob/d36752ab704eedfca6a53f447cdb3e5d7ca5bc77/src/gi-stubs/repository/_Gdk4.pyi#L2498)

The remaining methods, `fill`, `line_to` and `move_to` all exist in `Gsk.pyi`, `GskBase.pyi`, or `GdkPixbuf.pyi`, but they don't seem related.

### My Research, Background, and Thoughts

The [`Gdk.CairoContext`](https://api.pygobject.gnome.org/Gdk-4.0/class-CairoContext.html) is passed by a [`Gtk.DrawingArea`](https://api.pygobject.gnome.org/Gtk-4.0/class-DrawingArea.html) when it calls the `draw` method registered with the object.

Perhaps importantly, [`pycairo`](https://pypi.org/project/pycairo/) is a (the only) dependency for `PyGObject`. It claims to be a "Fully typed and documented API", the drawing methods I'm using _are_ documented at [Cairo Context](https://pycairo.readthedocs.io/en/latest/reference/context.html), and they _are not_ mentioned in the [GNOME Python API documentation](https://api.pygobject.gnome.org/index.html).

There are no mypy errors for this untested `cairo.Context` code (Interestingly, `pylint` can't find cairo.SVGSurface or cairo.Context for some reason):

```
import cairo
with cairo.SVGSurface("example.svg", 200, 200) as surface:
    context = cairo.Context(surface)
    context.move_to(1, 1)
```

I'm not familiar enough with how all of this works to draw any conclusions, but it makes me wonder if I'm actually using a `cairo.Context` object and it's being mistyped as `CairoContext`.

--- a/src/gi-stubs/repository/Gio.pyi
+++ b/src/gi-stubs/repository/Gio.pyi
@@ -13,7 +13,7 @@ from gi.repository import GLib
 from gi.repository import GObject
 
 T = TypeVar("T")
-ObjectItemType = TypeVar("ObjectItemType", bound=GObject.Object)
+ObjectItemType = TypeVar("ObjectItemType", bound=GObject.Object, default=Any)
 
 DBUS_METHOD_INVOCATION_HANDLED: bool = True
 DBUS_METHOD_INVOCATION_UNHANDLED: bool = False

